docs/Project-Configuration/Project-Commands.md

```diff
@@ -560,6 +560,9 @@ Notes:
 -   generate.tasks is idempotent across task restarts. In other words, once
     generate.tasks succeeds once, later task restarts will cause generate.tasks
     to no-op rather than generate duplicate configuration.
+-   The command does not give any feedback (via logs or the UI) what
+    tasks were generated so using [s3.put](#s3put) after the command to upload
+    the JSON file used in recommended for debuggability.
 
 ``` yaml
 - command: generate.tasks
```

Comment:
would it be possible to include an s3.put t hat would do this in this sample yaml?

Reply:
I added an example